Contents/Code/siteUltrafilmsxxx.py

```diff
@@ -0,0 +1,108 @@
+import PAsearchSites
+import PAgenres
+import PAactors
+import PAextras
+import ssl
+from lxml.html.soupparser import fromstring
+
+# maybe helpful for linux users, who has "sslv3 alert handshake failure (_ssl.c:590)>" @kamuk90
+def search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate,searchSiteID):
+    if searchSiteID != 9999:
+        siteNum = searchSiteID
+    url = PAsearchSites.getSearchSearchURL(siteNum) + encodedTitle
+    headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.47 Safari/537.36'}
+
+    try:
+        searchResults = HTML.ElementFromURL(url)
+    except:
+        request = urllib.Request(url, headers=headers)
+        response = urllib.urlopen(request, context=ssl.SSLContext(ssl.PROTOCOL_TLSv1))
+        htmlstring = response.read()
+        searchResults = fromstring(htmlstring)
+
+    for searchResult in searchResults.xpath('//ul[@class="listing-videos listing-tube"]'):
```

Comment:
thanks for the ssl workaround!
but the xpath dont work for me, try this:

`    for searchResult in searchResults.xpath('//*[div[@class="format-infos"]/div[@class="time-infos"]]'):
        titleNoFormatting = searchResult.xpath('.//a')[0].get('title').strip()
        Log("titleNoFormatting: " + titleNoFormatting)
        actors = searchResult.xpath('.//img')[0].get('alt')
        Log(actors)
        sceneUrl = searchResult.xpath('.//a')[0].get('href')
        curID = sceneUrl.replace('/','_').replace('?','!')
        Log("curID: " + curID)
        score = 100 - Util.LevenshteinDistance(searchTitle.lower(), titleNoFormatting.lower())
        results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = titleNoFormatting + " with "+ actors + " [" + PAsearchSites.getSearchSiteName(siteNum) + "] ", score = score, lang = lang))
    return results`